aclk/legacy/aclk_common.c

```diff
@@ -234,3 +234,26 @@ int aclk_decode_base_url(char *url, char **aclk_hostname, int *aclk_port)
     info("Setting ACLK target host=%s port=%d from %s", *aclk_hostname, *aclk_port, url);
     return 0;
 }
+
+struct label *add_aclk_host_labels(struct label *label) {
+    #ifdef ENABLE_ACLK
```

Comment:
The code is OK, but I suspect the current tabulation was not desired, please, can you confirm this?

Reply:
yes, thank you it was oversight on my side